RULES.md

```diff
@@ -16,21 +16,23 @@ These are the Rules for the All the hackathon Events and Related Things
 ----
 7. Everyone attending Should bring their laptops to Hack-on, no laptops or accessories will be provided by the Organizers.
 ----
-8. Those Who want to Hack-on Hardware, they should bring their own Hardware Equipment. If these components bring about any damage to the venue, They Will be charged for the Damages
+8. Those Who want to Hack-on Hardware, they should bring their own Hardware Equipment. If these components bring about any damage to the venue, They Will be charged for the Damages.
 ----
-9. If anyone Uses the Internet Connectivity to Download Films, Pornographic content. They will be disqualified and the consequences will be same as 2.1
+9. If anyone Uses the Internet Connectivity to Download Films, Pornographic content. They will be disqualified and the consequences will be same as 2.1.
 ----
-10. If anyone is found out be Harassing Others, especially Women Participants will also be Disqualified by 2.1
+10. If anyone is found out be Harassing Others, especially Women Participants will also be Disqualified by 2.1.
 ----
 11. Attendees should Bring their College/School ID card with necessary items of their Needs.
 ----
-12. If Attendees have any Dietary restrictions please send a request to meenhacks@gmail.com
+12. If Attendees have any Dietary restrictions please send a request to meenhacks@gmail.com.
 ----
-13. If any attendes require Stay before or After the Event, they should send a request to meenhacks@gmail.com
+13. If any attendes require Stay before or After the Event, they should send a request to meenhacks@gmail.com.
 ----
```

Comment:
Add this too `The certificates for hackathon and workshops will be given separately`